tests/DCPS/Compiler/vread_vwrite/VreadVwriteTest.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": 5,
+  "data": "The most rapid of JSONs",
+  "enu": "three",
+  "enu2": "two",
+  "bt": {
+    "o": 129,
+    "u8": 130,
+    "i8": -50,
+    "us": 32777,
+    "s": -32765,
+    "ul": 2147483690,
+    "l": -2147483632,
+    "ull": 9223372036854775810,
+    "ll": -9223372036854775806,
+    "f": 0.0014100000262260437,
+    "d": -0.00000141,
+    "ld": 1e+34,
+    "b": false,
+    "c": "\r",
+    "str": "The most JSON of rapids",
+    "wstr": "Τηισ ισ α τεστ οφ τηε εμεργενψυ βροαδψαστ συστεμ. פךקשדק לקקפ טםור ישמגד ןמדןגק איק הקיןבךק שא שךך אןצקד"
```

Comment:
I think we mentioned removing UTF-8 from our json files recently... is now the time to try to switch the test strings over to escaped unicode points?

Reply:
What was the motivation for that?

Reply:
The security issue with unicode-encoded source files. I believe there are escape sequences which can move the cursor, allowing for "rewriting" the string. @mitza-oci was the one who brought it up, originally.

Reply:
So the JSON might be fine, but the place where it's encoded in C++ wouldn't be, for example.

Reply:
This can probably wait...